www/source/partials/docs/_glossary-launcher.html.md.erb

```diff
@@ -0,0 +1,23 @@
+# <a name="glossary-launcher" id="glossary-launcher" data-magellan-target="glossary-launcher">Launcher</a>
+
+Habitat's Launcher is a sidecar process for the Supervisor which provides a mechanism for launching processes on behalf of the Supervisor. It is the entry point for running the Supervisor and is the Supervisor for the Supervisor. Whereas the Supervisor is able to automatically update itself, the Launcher is currently released a bit differently, by design; it should be rare that the Launcher ever needs to change.
```

Comment:
No, why would we leave this information out?

Reply:
To be clear: We want this information here because we need to communicate the design element here: the Launcher should be rare to change. This is important because it doesn't automatically self-update, and thus, the operator will need to take some action and pay attention to new Launcher versions.

Reply:
If you think it is a meaningful term for our users, the let's keep it.